content/help/events/RandRustBinding.yaml

```diff
@@ -0,0 +1,7 @@
+title: "R & Rust binding"
+location: "Virtual"
+event_host: "Developer forum"
+start: 2025-06-28
+end: 2025-06-28
```

Comment:
This date is using June instead of July as the month it will be held...

Reply:
Apologies, I mistakenly wrote the wrong month. I’ll submit a corrected pull request.